src/diff/diffCommand.test.ts

```diff
@@ -111,6 +111,13 @@ const operationRemovedRuleset = createTempFile(
 );
 
 describe("raml-toolkit cli diff command", () => {
+  let baseDir: DirResult;
```

Comment:
Now that we have better output for errors, maybe adjust tests to check that output as well, instead of just exit code.